### ISSUE TITLE
Overworld rendering broken in 1.18

### ISSUE THREAD

**Issue by DocOfPineapples** [papyrus-mc/papyruscs#93]
This morning I tried running papyrus in Minecraft 1.18. However, it seems this patch has broken the rendering for papyrus as new and some old areas are rendered as if they were underground (below is a picture of what happens). I'm running papyrus on ubuntu server 20.04 and I'm using the latest commit from the master branch (the 1.17.30 chunk fix). I don't know enough about this project to try and contribute to this code but I figured this would be the better place to post this issue instead of the discord. Let me know if there is any other information I should be providing. 

![Untitled](https://user-images.githubusercontent.com/84910898/144128918-1c8effff-76a2-46ca-9b26-6227394ee752.jpg)

**Comment by TheDogg** [comment]
I can confirm I have the same issue.

**Comment by dopazz** [comment]
I have the same problem.  As a workaround I added two switches to the command line:

--notrimceiling --limity 200

This output a proper rendering of the surface for me.

**Comment by DocOfPineapples** [comment]
I tested it out and I eventually got it to work with --notrimceiling --y 319. I used 319 instead of 200 since it is the new build height for 1.18.

Hopefully, someone will be able to modify the source so it works correctly with 1.18 soon so we don't have to rely on this.

**Comment by dustinosity** [comment]
@DocOfPineapples can I ask what the full command you use to generate is? I just tried to use this yesterday on an 1.18 world and all I'm getting is blank image tiles. thinking I have a more than one issue at once here...

**Comment by dustinosity** [comment]
> @DocOfPineapples can I ask what the full command you use to generate is? I just tried to use this yesterday on an 1.18 world and all I'm getting is blank image tiles. thinking I have a more than one issue at once here...

ok, some progress: just built master from source and that helped a bit. rendering non blank chunks now. just has some weird missing blocks.

**Comment by dustinosity** [comment]
> > @DocOfPineapples can I ask what the full command you use to generate is? I just tried to use this yesterday on an 1.18 world and all I'm getting is blank image tiles. thinking I have a more than one issue at once here...
> 
> ok, some progress: just built master from source and that helped a bit. rendering non blank chunks now. just has some weird missing blocks.

ok, i just forced an overwrite of everything and now it's not so bad.

**Comment by DocOfPineapples** [comment]
Hi @dustinosity, yeah I had some issues too when I first tried to get it working and also assumed the problems I had it is related to another problem I might be happening. Here is the command that eventually worked: 

PapyrusCs --world WorldLocation --output OutputLocation --htmlfile index.html --playericons true --deleteexistingupdatefolder true -d 0 --notrimceiling true -y 319 --forceoverwrite

After I got a fully render I stopped using the --forceoverwrite, I still need to check if new chunks generated after the reader come through properly. But, the first time I did it it didn't fully render everything. I think the --forceoverwrite helped fix that but it could have been something else and I don't remember exactly what I did. But it sounds like you got it working.
